plugins/clipboard/plugin.js

```diff
@@ -3002,6 +3123,16 @@
  * @member CKEDITOR.editor
  */
 
+/**
+ * Internal event to open the Paste dialog window.
+ *
+ * @private
+ * @event pasteDialog
```

Comment:
We should note that [`pasteDialog` event](https://github.com/ckeditor/ckeditor-dev/blob/34a59d1/plugins/clipboard/plugin.js#L3130) was not available in 4.7.0 - 4.8.0 versions.